functions/packages/feed-form/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import {initializeApp} from "firebase-admin/app";
+import {onRequest} from "firebase-functions/v2/https";
+import * as feedAPI from "./impl/feed-form-impl";
+import {defineSecret} from "firebase-functions/params";
+const googleSheetSpreadsheetId = defineSecret("GOOGLE_SHEET_SPREADSHEET_ID");
```

Comment:
How are we deploying these secrets?

Reply:
I deployed them from my terminal ```npx firebase functions:secrets:set GOOGLE_SHEET_SERVICE_EMAIL``` I set them for all environments dev, qa, prod

Reply:
We need to move this process to GH actions and avoid deployments from the local machines.

Reply:
I wonder if there is an alternative way to use the cloud's function service account and omit the private key.

Reply:
These secrets are a 'one and done' set mechanic, similar to like setting environment variables in Github. They don't need to be set on every deployment

Reply:
Looking at https://theoephraim.github.io/node-google-spreadsheet/#/guides/authentication we could create an API Key instead of Account Service Email and Private Key

Reply:
I understand, but always avoid depending on local machines for prod deployment. This process is not scalable and requires setting up local environments, which can lead to errors. When keys change or are exposed, we need to redeploy them. Having them in a centralized place like 1Password help us to do this maintenance task.